auth/api/iam/api.go

```diff
@@ -362,7 +369,14 @@ func (r Wrapper) HandleAuthorizeRequest(ctx context.Context, request HandleAutho
 	case responseTypeVPToken:
 		// Options:
 		// - OpenID4VP flow, vp_token is sent in Authorization Response
-		return r.handleAuthorizeRequestFromVerifier(ctx, *ownDID, params)
+		// non-spec: wallet_owner_type is an unsigned parameter that hints whether the request targets an organization or user wallet.
+		// Requests to user wallets can be rendered as QR-code.
+		walletOwnerType := pe.WalletOwnerType(params.get("wallet_owner_type"))
```

Comment:
In the initial design we talked about handling all urls beginning with `openid4vp://` as user target. This switch to a specific param, is it registered somewhere (ticket)?

Reply:
Changed it to internal "dispatch". Feels kinda sketchy just calling the HTTP function, but let's see if it works.